CMakeLists.txt

```diff
@@ -7,12 +7,6 @@ set(CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/CMakeModules)
 
 enable_testing()
 
-# check for Qt
-find_package(Qt5Core)
-find_package(Qt5Gui)
-find_package(Qt5Widgets)
-find_package(Qt5OpenGL)
-
 # check for SDL2
 find_package(SDL2)
```

Comment:
For the sake of consistency, if the checks for Qt are removed from the top-level, this check for SDL2 should as well, and be put in `examples/playground`.

Reply:
I will move back the Qt5 check to the top-level file, it makes more sense